tendrl/alerting/handlers/cluster/brick_utilization_handler.py

```diff
@@ -0,0 +1,28 @@
+from tendrl.alerting.handlers import AlertHandler
+from tendrl.alerting.objects.cluster_alert import ClusterAlert
+
+
+class BrickUtilizationHandler(AlertHandler):
+
+    handles = 'brick_utilization'
+    representive_name = 'brick_utilization_alert'
+
+    def __init__(self):
+        AlertHandler.__init__(self)
```

Comment:
May 26 04:11:30 dhcpxx-xxx tendrl-alerting: super(BrickUtilizationHandler, self).__init__()
May 26 04:11:30 dhcpxx-xxx tendrl-alerting: NameError: global name 'BrickUtilizationHandler' is not defined